tide/master/MultitouchArea.cpp

```diff
@@ -126,7 +127,7 @@ void MultitouchArea::mousePressEvent( QMouseEvent* mouse )
      _mousePrevPos = pos;
      emit touchStarted( pos );
 
-     _startTapAndHoldGesture();
+     _startTapAndHoldGesture( Positions{ pos } );
```

Comment:
You don't need the Positions here ? _startTapAndHoldGesture( { pos } ); should be fine.